### DISCUSSION TITLE
Missing "Hide this field" in Logs Panel

### DISCUSSION THREAD

**Discussion by thies88** [grafana/grafana#46280] [Observability]
**What happened**: Missing "Hide this field" in Logs panel

**What you expected to happen**: have the option "Hide this field"

**How to reproduce it (as minimally and precisely as possible)**: 

Using Explore: 
![afbeelding](https://user-images.githubusercontent.com/22106095/156881235-ba7ab89b-b702-44fb-af66-1287a73515bd.png)

Using Logs Panel:
![afbeelding](https://user-images.githubusercontent.com/22106095/156881210-37181c72-76a6-4f7e-876b-698900295eec.png)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version: Loki: v2.4.2
- OS Grafana is installed on: Ubuntu 20.04 core (Docker)
- User OS & Browser: Windows 10: FireFox
- Grafana plugins: Panel Logs
- Others:

**Comment by 09jvilla** [comment]
Hey @thies88 -- Is this something you're saying you're just noticing recently? As far as I know, this is how the experience has been for a while. That being said, I understand why it might be useful to be able to have that same 'hide this field' functionality for Log labels. 

Can you say a little bit more about your specific use case and why you like the 'hide this field' button / how you use it in your workflow? 

cc @mary-sitzenstatter @ivanahuckova @niathadgu

**Comment by ivanahuckova** [comment]
This is correct behaviour. We don't have `filter for` and `filter out` functionalities implemented in Logs panel. We need to discuss with UX (@mary-sitzenstatter @niathadgu) what should be the behaviour when you click on `filter for` or `filter out`. Should we redirect to Explore? Should we change the query in dashboard with some visual indication that query was changed? Should we change the query in dashboard without any visual indication that query was changed? Is it possible to save this new query without going to edit mode?

**Comment by ivanahuckova** [comment]
As this is a feature request, I am going to change it to discussion.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
